modules/nw-ne-openshift-dns.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+// * assembly/networking
+
+[id='networking-ne-openshift-dns-{context}']
+= {product-title} DNS
+
+If you are running multiple services, such as frontend and backend services for
```

Comment:
front-end and back-end should be hyphenated since they're adjectives (throughout the file).